### ISSUE TITLE
necesitamos incorporar styles-es para cambiar el aspecto del sitio web.

### ISSUE THREAD

**Issue by raynamharris** [Carpentries-ES/python-ecology-lesson-es#40]
para cambiar el aspecto del sitio web de este
![screen shot 2018-11-14 at 4 38 04 pm](https://user-images.githubusercontent.com/7544197/48517373-b902b400-e82b-11e8-8528-e843bd3f0767.png)

a esto

![screen shot 2018-11-14 at 4 37 49 pm](https://user-images.githubusercontent.com/7544197/48517376-bb650e00-e82b-11e8-8a59-800af509fc74.png)

necesitamos incorporate  el repositorio [styles-es](https://github.com/carpentries/styles-es). 
Talvez @fmichonneau o @rgaiacs se puede ayudar

**Comment by orchid00** [comment]
@raynamharris 
Two comments
Epidodios
Mejora esta página

**Comment by raynamharris** [comment]
good catch! suggestions fixed in https://github.com/carpentries/styles-es/pull/13

**Comment by fergarciafer** [comment]
Why not "Code of Conduct" with "Código de Conducta"?

**Comment by raynamharris** [comment]
Hi @fergarciafer. I wondered the same thing at first, but then I figured it out. 

Currently, I have a Spanish translation of the code of conduct in our repo, so "Código de Conducta" does make sense. However, both the [styles](https://github.com/carpentries/styles) and [styles-es](https://github.com/carpentries/styles-es) repository (which is where all the formatting is set for the look and feel of lessons) the code of conduct page serves merely as a placeholder and links to the official [Code of Conduct on The Carpentries website](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html). So, this means that having the "Code of Conduct" makes sense because it is in English. 

I wonder if it would make sense to propose a hybrid solution: We could translate the Code of Conduct into Spanish but also say, something like, "the English version is the definitive version" and provide the link.

**Comment by fergarciafer** [comment]
Hi @raynamharris .

I think it's a good idea. I have already finished with the issue I was assigned to, and now I am able to continue collaborating on your proposal or any other activity.

**Comment by raynamharris** [comment]
Thanks, @fergarciafer. This is something we can put on hold until we are completely done with the translation. I also usually need to request help from staff members because the process can be a little tricky even though it is [explained here](http://carpentries.github.io/lesson-example/setup.html))

**Comment by orchid00** [comment]
hi @raynamharris, Last PR has just been merged! :tada: 

how do we go about incorporating the styles-es and publishing the lesson?
Maybe @ErinBecker can help? please also see the standing PR in the styles-es repo https://github.com/carpentries/styles-es/pull/13 before incoorporating

**Comment by raynamharris** [comment]
Woohoo! I link @rgaiacs helped me with this in the past...

**Comment by rgaiacs** [comment]
@raynamharris 

```
$ git clone git@github.com:Carpentries-ES/python-ecology-lesson-es.git
$ cd python-ecology-lesson-es
$ git remote add styles git@github.com:carpentries/styles-es.git
$ git fetch styles
$ git merge styles/gh-pages
$ git push origin gh-pages
```

**Comment by orchid00** [comment]
Thanks @rgaiacs ! I've followed the instructions and now we have styles-es :+1: 

What is next? @raynamharris

**Comment by raynamharris** [comment]
woohoo! ya estamos listo!!

**Comment by orchid00** [comment]
linking to this PR https://github.com/Carpentries-ES/python-ecology-lesson-es/pull/59/files 
but don't know how to proceed, I've asked @fmichonneau to review the PR

**Comment by raynamharris** [comment]
yea! fixed with #59. Thanks @jmbarrios
